illumidesk/templates/alb-rbac-role.yaml

```diff
@@ -6,8 +6,13 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   labels:
+<<<<<<< HEAD
```

Comment:
Looks like you committed som tags from a local merge/rebase - been there done that! 😄